### ISSUE TITLE
Allow Easy Posts to return taxonomies on any page

### ISSUE THREAD

**Issue by Gibsonion** [soflyy/oxygen-bugs-and-features#209]
One limitation with Easy Posts is not being able to display taxonomies on most pages. I can display the content in taxonomies, but not the taxonomies themselves.

Uses:
Displaying lists of available categories or tags anywhere on the site.

I use Easy posts to display woocommerce products, but would love to be able to display available shopping categories, or tags sitewide that displays the relevant image and metadata dynamically.

Eamples:
Many plugins offer this feature on the WordPress plugin library, but none do a good job.

**Comment by LisiLeap** [comment]
Adding: Tag, Category and Archive post results are from Wordpress too which are second rate (look ugly and are limited with what you can do with it). It would be nice to be able to edit and format these taxonomies globally with Oxygen. Many Travel and Lifestyle bloggers use these so Oxygen would get more buyers if this were enhanced. A design set for a Travel Blogger heavy on Taxonomies would also lure more bloggers over. Just saying.

**Comment by Gibsonion** [comment]
Yeah totally at the same time I'm struggling to understand why building archive pages relies on using Easy Posts to implement the loop when simply implementing the exact same query controls directly into any pages marked as arrive would instantly solve /eliminate the need for about about 20 different feature requests, Visual Easy Posts, Visual WooCommerce page builder, Extending Code Blocks, More Blog Templates to name a few

**Comment by LisiLeap** [comment]
Adding one more taxonomy issue where the solution or wish would be to have them completely customizable in Oxygen. Categories come in trees or hierarchies. Example: North America>USA>New York>Niagara Falls. Problem: Travel Bloggers write about areas. Niagara Falls spans across two countries so a second Category hierarchy should be applied to the post too: North America>Canada>Ontario>Niagara Falls. Categories are run by Wordpress which is limited: only one hierarchy allowed. If Oxygen made this customizable where you can add unlimited hierarchies, it would be another bonus. Many other parks throughout the world are like this by the way. thanks
